nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/slick-nifi-theme.css

```diff
@@ -126,7 +129,7 @@
     border-top: 0;
     border-left: 1px solid #c7d2d7;
     border-bottom: 1px solid #c7d2d7;
-    padding: 0 2px 6px 10px;
+    padding: 1 2px 5px 10px;
```

Comment:
It looks like your padding-top property needs a unit of measurement (ex. 1px).

Reply:
Good catch. I've added 'px' and pushed the change.